### DISCUSSION TITLE
API 명세서에 대해

### DISCUSSION THREAD

**Discussion by laigasus** [Strong-Potato/TripVote-BE#52] [General]
뭐로 작성해서 프론트분들에게 드려야 할까요?

**Comment by happymink** [comment]
![image](https://github.com/Strong-Potato/TripVote-BE/assets/71982555/3514bbb4-62a3-46a4-b95e-5401f4904876)

저는 임시로 노션에 작성할 땐 이렇게 했었습니다! 요청방식과 응답방식은 각 항목의 상세 페이지에 작성했어요

**Comment by laigasus** [comment]
2024-01-11 회의에서
swaggerhub를 통해 배포하기로 정했음
단. 일정 기간 내 동작하지 않으면 notion에 작성될 예정

https://github.com/Strong-Potato/TripVote-BE/issues/53
